code/modules/client/preference_setup/loadout/items/shoes.dm

```diff
@@ -164,6 +164,10 @@
 	recolourable_shoes["oxfords"] = /obj/item/clothing/shoes/laceup/colourable
 	gear_tweaks += new /datum/gear_tweak/path(recolourable_shoes)
 
+/datum/gear/shoes/falerio_boots
```

Comment:
should be in boots selection, and not a separate item

Reply:
Moved it into the jackboot section, I think?